src/test/java/com/razorpay/DisputeClientTest.java

```diff
@@ -0,0 +1,246 @@
+package com.razorpay;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+
+import java.io.IOException;
+import java.util.List;
+import static org.junit.Assert.*;
+
+public class DisputeClientTest extends BaseTest{
+
+    @InjectMocks
+    protected DisputeClient disputeClient = new DisputeClient(TEST_SECRET_KEY);
+
+    private static final String DISPUTE_ID = "disp_AHfqOvkldwsbqt";
+
+    /**
+     * Contest a Dispute using dispute id.
+     * @throws RazorpayException
+     */
+    @Test
+    public void TestContest() throws RazorpayException{
+
+        JSONObject request = new JSONObject("{\n" +
+                "  \"amount\": 5000,\n" +
```

Comment:
As passing a string to JSONObject looks untidy, please do it the same way it is done here https://github.com/razorpay/razorpay-java/blob/55a19372e8aeb9c74cbdabd8286356252a0304a2/src/test/java/com/razorpay/CustomerClientTest.java#L308